tests/stability/test_spill.py

```diff
@@ -6,42 +6,15 @@
 from dask.distributed import Client, wait
 
 
-@pytest.mark.stability
-@pytest.mark.parametrize("keep_around", (True, False))
-def test_spilling(keep_around, upload_cluster_dump, benchmark_all):
+@pytest.fixture(scope="module")
+def spill_cluster():
     with Cluster(
-        name=f"test_spill-{uuid.uuid4().hex}",
+        f"spill-{uuid.uuid4().hex[:8]}",
         n_workers=5,
-        worker_disk_size=55,
-        worker_vm_types=["t3.medium"],
-        wait_for_workers=True,
-        environ={
-            # Note: We set allowed-failures to ensure that no tasks are not retried
-            #  upon ungraceful shutdown behavior during adaptive scaling
-            #  but we receive a KilledWorker() instead.
-            "DASK_DISTRIBUTED__SCHEDULER__ALLOWED_FAILURES": "0"
-        },
-    ) as cluster:
-        with Client(cluster) as client:
-            with upload_cluster_dump(client, cluster), benchmark_all(client):
-                arr = da.random.random((200, 2**27)).persist()  # 200 GiB
-                wait(arr)
-                fut = client.compute(arr.sum())
-                if not keep_around:
-                    del arr
-                wait(fut)
-
-
-@pytest.mark.skip(
-    reason="Skip until https://github.com/coiled/feedback/issues/185 is resolved."
-)
-@pytest.mark.stability
-def test_tensordot_stress(upload_cluster_dump, benchmark_all):
-    with Cluster(
-        name=f"test_spill-{uuid.uuid4().hex}",
-        n_workers=5,
-        worker_disk_size=55,
-        worker_vm_types=["t3.medium"],
+        package_sync=True,
+        worker_disk_size=64,
+        worker_vm_types=["t3.large"],
+        scheduler_vm_types=["t3.large"],
```

Comment:
@hendrikmakait in #280 you seemed okay with making the instances here a bit beefier. Is that still the case?